libs/designsystem/src/lib/components/calendar/calendar.component.scss

```diff
@@ -7,27 +7,37 @@ table.calendar-table {
   user-select: none;
   outline: none !important;
   border-spacing: 0;
+  margin-bottom: size('xxs');
 
   thead {
-    padding: 0.5em;
     border-bottom: 1px solid get-color('background-color');
 
     th {
       height: 50px;
       text-align: center;
-    }
-  }
 
-  tbody {
-    padding: 0.5em;
+      &:first-child {
+        padding-left: size('xxs');
+      }
+
+      &:last-child {
+        padding-right: size('xxs');
+      }
+    }
   }
 
   tr,
   td {
     border-style: none;
     padding: 0;
-    line-height: 2em;
-    height: 0;
+    text-align: center;
+
+    td:first-child {
+      padding-left: size('xxs');
+    }
+    td:last-child {
+      padding-right: size('xxs');
+    }
```

Comment:
These nested selectors are a bit odd, since you don't have nested tables.

I suggest you remove the outer `tr` selector and replace the `td` in `td:first-child` and `td:last-child` with an ampersand `&`